Easydict/App/Localizable.xcstrings

```diff
@@ -4377,6 +4394,202 @@
         }
       }
     },
+    "tips_beep" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Why is there a beep when the selected word is empty (such as dragging in a blank area) in some applications?"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "为什么在某些应用取词为空（空白处拖动等）时会出现提示音？"
```

Comment:
After PR https://github.com/tisfeng/Easydict/pull/539 , this bug has fixed, so we don't need this tip.